lib/core-server/src/utils/stories-json.ts

```diff
@@ -52,7 +71,20 @@ async function extractStories(normalizedStories: NormalizedStoriesSpecifier[], c
       }
     })
   );
-  return stories;
+
+  const previewFile = ['js', 'jsx', 'ts', 'tsx']
+    .map((ext) => path.join(configDir, `preview.${ext}`))
+    .find((fname) => fs.existsSync(fname));
```

Comment:
I guess we aren't going to have back-compat for `config.js`?

In theory is it possible for an addon to add its own story-sort parameter. Are we not allowing this?

Reply:
No and no.